galaxy_ng/app/api/ui_v2/serializers.py

```diff
@@ -41,16 +43,19 @@ def get_groups(self, obj):
         return groups_serializer.data
 
     def get_teams(self, obj):
-        teams = Team.objects.filter(users=obj)
+        """Return all 'local' and 'non-local' team member assignments."""
+        roledefs = RoleDefinition.objects.filter(
+            name__icontains='Team Member'
```

Comment:
Because @AlanCoding wants this to return "local" and "non-local" memberships, and I could provide a list of roledef PKs, but icontains also sufficed.

Reply:
Thinking about it more ... it could be conceived that user might make their own custom role definition with a name like "Jimbob Team Member" and that would mess up this logic. I'll change the code to be more "strict".